src/index.js

```diff
@@ -77,20 +80,35 @@ const handleHideAll = (target, groupTarget, toggleClass) => {
 
 const setupToggle = () => {
 
+    /**
+     * If accordion, only display first section on initialisation
+     */
+
+    $('[data-toggle-accordion]')
+        .forEach(accordion => {
+            const toggleClass = accordion.getAttribute('data-toggle-class') || 'is-hidden';
+            const sections = [...accordion.querySelectorAll('[data-toggle-name]')];
+
+            sections.splice(0, 1);
```

Comment:
Prefer using array methods which don't change the array in place, so in this case you could use `slice` like so: 

```js
sections
    .slice(0, 1)
    .forEach(toggles(toggleClass).hide);
```

Reply:
Won't this do a slightly different thing?

If you're just removing the first thing from the array, it would be

```
sections.slice(1)
.forEach(…)
```

Wouldn't it?

Reply:
Sorry, yes, you're right @ashleynolan